akka-remote/src/main/scala/akka/remote/serialization/MiscMessageSerializer.scala

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.remote.serialization
+
+import akka.actor._
+import akka.protobuf.ByteString
+import akka.remote.ContainerFormats
+import akka.serialization.{ BaseSerializer, SerializationExtension, SerializerWithStringManifest }
+
+class MiscMessageSerializer(val system: ExtendedActorSystem) extends SerializerWithStringManifest with BaseSerializer {
+
+  @deprecated("Use constructor with ExtendedActorSystem", "2.4")
+  def this() = this(null)
```

Comment:
this is not needed, it was kept in the other for backwards compatibility reasons